apps/api/src/recurring-donation/recurring-donation.service.ts

```diff
@@ -18,9 +29,103 @@ export class RecurringDonationService {
     return await this.prisma.recurringDonation.findMany()
   }
 
+  static defaultSelectFields = {
```

Comment:
You can use `Prisma.validate()` to better type this field.

https://www.prisma.io/docs/concepts/components/prisma-client/advanced-type-safety/prisma-validator

You can also use `includes` key instead of `select` key to skip mentioning every single field in the initial model and only specify the relations like `sourceVault` and `person`_